src/Tests/Microsoft.Diagnostics.Monitoring.StartupHook.UnitTests/Exceptions/Eventing/ExceptionsEventSourceTests.cs

```diff
@@ -78,16 +78,22 @@ public void ExceptionsEventSource_WriteException_Event(ulong id, string message,
 
             ulong[] frameIds = frameIdsString.Split(',', StringSplitOptions.RemoveEmptyEntries).Select(ulong.Parse).ToArray();
             DateTime timestamp = DateTime.UtcNow;
+            ulong[] innerExceptionIds = innerExceptionIdsString.Split(',', StringSplitOptions.RemoveEmptyEntries).Select(ulong.Parse).ToArray();
```

Comment:
Is this being used?

Reply:
Good catch. I missed passing it in on line 83.